examples/platforms/gp712/radio.c

```diff
@@ -196,7 +197,24 @@ otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aPacket)
 
 void cbQorvoRadioTransmitDone(otRadioFrame *aPacket, bool aFramePending, otError aError)
 {
-    otPlatRadioTransmitDone(pQorvoInstance, aPacket, aFramePending, aError);
+    otRadioFrame ackFrame;
+    uint8_t psdu[IEEE802154_ACK_LENGTH];
+
+    ackFrame.mPsdu = psdu;
+    ackFrame.mLength = IEEE802154_ACK_LENGTH;
+    ackFrame.mPsdu[0] = IEEE802154_FRAME_TYPE_ACK;
+
+    if (aFramePending)
+    {
+        ackFrame.mPsdu[0] |= IEEE802154_FRAME_PENDING;
+    }
+
+    ackFrame.mPsdu[1] = 0;
+    ackFrame.mPsdu[2] = aPacket->mPsdu[IEEE802154_DSN_OFFSET];
+
+    otRadioFrameFillCRC(&ackFrame);
```

Comment:
This should not be necessary.  OpenThread does not use the IEEE 802.15.4 FCS field.